xchange-coinbasepro/src/main/java/org/knowm/xchange/coinbasepro/service/CoinbaseProMarketDataServiceRaw.java

```diff
@@ -50,6 +55,24 @@ public CoinbaseProProductStats getCoinbaseProProductStats(CurrencyPair currencyP
     }
   }
 
+  public List<CoinbaseProStats> getCoinbaseProStats() throws IOException {
+    try {
+      Map<String, CoinbaseProStats> coinbaseProStats = coinbasePro.getStats();
+      Set<String> pairs = coinbaseProStats.keySet();
+      List<CoinbaseProStats> coinbaseProStatsList = new LinkedList<>();
+      for (String pair : pairs) {
+        String[] pairSplit = pair.split("-");
+        CoinbaseProStats stats = coinbaseProStats.get(pair);
+        stats.setCurrencyPair(new CurrencyPair(pairSplit[0], pairSplit[1]));
```

Comment:
this will always modify the objects in coinbasePro

please ma objects coinbasePro to already have this property set